minio/minio
# MQTT Persistence for non-registered events

Persist events even when the client has not registered the token.

## Description
The client has to register and obtain a valid token atleast once to make the events persist.

## Motivation and Context
The events are lost if minio is started without starting the mqtt broker

## Regression
No

## How Has This Been Tested?

- Do not start the mqtt broker yet.
- Start minio with ARN and queueDir configured - we can see the ARN registered even when the mqtt broker is not yet started ( Which was responding with ARN Not found error before)
- Start sending events 

We can see the events persisting in the queueDir even if the broker is not started yet. Now, if we start the broker. We can see the events getting queued. ( We can see it in activeMQ webconsole `http://localhost:8161/admin/topics.jsp`)


## Types of changes


## Checklist:
